src/Exceptions/InvalidVersionException.php

```diff
@@ -4,6 +4,9 @@
 
 use Exception;
 
+/**
+ * Exception that is thrown when an invalid semantic version string is used.
+ */
```

Comment:
I'd prefer to omit unnecessary doc blocks and comments.

```suggestion
```

Reply:
I think it’s bad practice not to PHPDoc (public) classes, traits, interfaces, functions. You have to decide. 🙃